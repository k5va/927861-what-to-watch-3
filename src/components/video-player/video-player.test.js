import {VideoPlayer} from "@components";

const VIDEO_SRC = `video`;
const MOVIE_COVER = `cover`;

it(`VideoPlayer should render correctly`, () => {
  const wrapper = renderer
    .create(
        <VideoPlayer src={VIDEO_SRC} poster={MOVIE_COVER} isPlaying={true} />,
        {
          createNodeMock: () => ({})
        }
    )
    .toJSON();
  expect(wrapper).toMatchSnapshot();
});
