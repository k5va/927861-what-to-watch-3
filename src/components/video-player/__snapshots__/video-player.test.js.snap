// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer should render correctly 1`] = `
<video
  height="175"
  width="280"
/>
`;
